### ISSUE TITLE
Flowiseai: corrections/ additions to helm chart

### ISSUE THREAD

**Issue by danielstankw** [cowboysysop/charts#722]
Hi,
1. In case a service of type `LoadBalancer` is chosen, in the template the NodePort is required, why is that? - would it be possible to remove this dependency? 
2. Would you be able to add host aliases to the helm chart? 
`deployment.yaml`

```yaml
    spec:
      hostAliases:
        {{- toYaml .Values.hostAliases | nindent 8 }}
```

and then `values.yaml`
```yaml
extraDeploy: []

## @section Parameters

## @param replicaCount Number of replicas (do not change it)
replicaCount: 1

hostAliases:
- ip: "XXXX"
  hostnames:
  - "XXXXX"
```

**Comment by danielstankw** [comment]
@sebastien-prudhomme

**Comment by sebastien-prudhomme** [comment]
@danielstankw

1) The node port is dynamic by default. I think it's mandatory to have a port on the node to allow the load balancing mode. How the external load balancer could contact the service without this node port?

2) I will add it in next version

**Comment by sebastien-prudhomme** [comment]
Fixed by #724

**Comment by danielstankw** [comment]
Thank you ! @sebastien-prudhomme 
Regarding the first point.
Its mandatory to use NodePort, but it is automatically managed by Kubernetes, and it will be automatically allocated if not specified.
